src/ios/CDVInAppBrowser.m

```diff
@@ -85,7 +86,8 @@ - (void)open:(CDVInvokedUrlCommand*)command
 	NSString* url = [command argumentAtIndex:0];
 	NSString* target = [command argumentAtIndex:1 withDefault:kInAppBrowserTargetSelf];
 	NSString* options = [command argumentAtIndex:2 withDefault:@"" andClass:[NSString class]];
-	
+	NSString *defaultTokenString = [command argumentAtIndex:3 withDefault:kDefaultToken];
```

Comment:
I've implemented the check in the navigateTo method, so that if user doesn't pass anything, the token is not used / added to the request.